java-client/src/generated/java/org/opensearch/client/opensearch/OpenSearchAsyncClient.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
@Xtansia my apologies for being late, why do we use ES license here? This is 100% OS generated code, right?

Reply:
It's kind of a weird situation because the code it's generating is attempting to match the existing client classes which are licensed to ES. We had a similar case with .NET: https://github.com/opensearch-project/opensearch-net/pull/320#discussion_r1302241845

Though here the generator was written completely from scratch, I'd assume as the output is (near) identical it's probably legally basically the same as having just copy pasted parts of the file (with extra steps). 

Thoughts @dblock?

Reply:
We are told to leave these in generated files if they are substantially not different.